enhancements/kube-apiserver/kms-encryption-provider.md

```diff
@@ -0,0 +1,131 @@
+---
+title: kms-encryption-provider
+authors:
+  - "@swghosh"
+reviewers:
+  - "@dgrisonnet"
+  - "@TrilokGeer"
+  - "@tkashem"
+  - "@rvanderp"
+approvers:
+  - ""
```

Comment:
@tkashem should be the default approver for anything api related as the team lead

You can add either myself or @deads2k for API approvers. But you need one of us. I've already looked at the API so probably me on this occasion.

@tkashem please review this EP and let me know that you're happy conceptually, and then I'll start reviewing from an API perspective

We should only merge the [feature gate PR](https://github.com/openshift/api/pull/2071#pullrequestreview-2393171997 once this has had an initial round of review

Reply:
Will add more details on this EP so we can have it ready for the initial round of review.